src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+
+const listContainer = document.querySelector('.list-Container');
+
+const tasklist = [
+  {
+    description: 'drink water',
+    completed: 'true',
+    index: 1,
+  },
+  {
+    description: 'go to hospital',
+    completed: 'false',
+    index: 2,
+  },
+  {
+    description: 'eat healthy',
+    completed: 'true',
+    index: 3,
+  },
+  {
+    description: 'do you like strawberries',
+    completed: 'false',
+    index: 4,
+  },
+  {
+    description: 'Dont forget to meditate ',
+    completed: 'true',
+    index: 5,
+  },
+  {
+    description: 'make sure your disturn BIN ',
+    completed: 'true',
+    index: 5,
+  },
+];
```

Comment:
- [ ] Here I can see you have used `index: 5` for both `object` but it should be different, cause as per the requirements `The tasks should appear in order of the index values for each task.` Kindly fix this issue.